de.finnik.PassVault.yml

```diff
@@ -0,0 +1,36 @@
+app-id: de.finnik.PassVault
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: passvault
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: passvault
+    buildsystem: simple
+    build-commands:
+      - install -D passvault /app/bin/passvault
+      - install -D PassVault-3.0.jar /app/share/PassVault-3.0.jar
+      - install -D de.finnik.PassVault.desktop /app/share/applications/de.finnik.PassVault.desktop
+      - install -D icon.svg /app/share/icons/hicolor/scalable/apps/de.finnik.PassVault.svg
+      - install -D de.finnik.PassVault.metainfo.xml /app/share/appdata/de.finnik.PassVault.metainfo.xml
+    sources:
+      - type: file
+        path: passvault
+      - type: file
+        path: PassVault-3.0.jar 
+      - type: file
+        path: de.finnik.PassVault.desktop
+      - type: file
+        path: icon.svg
+      - type: file
+        path: de.finnik.PassVault.metainfo.xml
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
```

Comment:
Ideally it should use `$XDG_DATA_HOME` to store its database.

Reply:
The idea behind the filesystem containing the whole home directory is that PassVault allows the user to export his passwords into a file and I'd like the user to be able to choose at least the directories inside his home directory